osu.Game/Graphics/UserInterface/OsuMenuItem.cs

```diff
@@ -19,6 +22,9 @@ public OsuMenuItem(string text, MenuItemType type, Action action)
             : base(text, action)
         {
             Type = type;
+
+            Enabled.BindValueChanged(enabled => Action.Disabled = !enabled.NewValue);
+            Action.BindDisabledChanged(disabled => Enabled.Value = !disabled);
```

Comment:
This gave me pause for a second. It's *correct*, but the fact of having both `Disabled` and `Enabled` made me re-read and make sure it's mapped correct. Would have been nice to have consistency (so `Disabled` everywhere or `Enabled` everywhere) but from what I see `ClickableContainer` *et al.* already have `Enabled` so that ship has probably already sailed.

Reply:
I find the outwards propagation a bit weird here. The action changing enabled state could potentially update the EditorChangeHandler's bindable. Maybe this should be a one-way thing instead? ie. the second line shouldn't exist (or at very least shouldn't propagate outwards).

Reply:
I added the bindable for the sole purpose of making it simpler to bind the event. I agree that it lead to unsafe scenarios so I've removed it since `Action.Disabled` can be used to promote one-way disabling.